csi/crypto/crypto.go

```diff
@@ -110,6 +110,18 @@ func EncryptVolume(devicePath, passphrase string, cryptoParams *EncryptParams) e
 		lhtypes.LuksTimeout); err != nil {
 		return errors.Wrapf(err, "failed to encrypt device %s with LUKS", devicePath)
 	}
+
+	isEncrypted, err = isDeviceEncrypted(devicePath)
```

Comment:
so it might not be encrypted after the format command, right?

Reply:
I am not sure. I added this for logging to bring in visibility in the support-bundle.

Reply:
The check looks good to me. We can keep it here.